packages/Select/src/select.scss

```diff
@@ -106,22 +113,27 @@
         border-color: var(--ids-select-border-color-hover);
     }
 
-    &:focus,
+    &:focus {
+        box-shadow: none;
+    }
+
+    &:focus-visible,
     &.focus {
-        outline: 0;
+        background-color: var(--ids-select-background-focus);
         box-shadow: var(--ids-select-focus);
         border-color: var(--ids-select-border-color-focus);
-        background-color: var(--ids-select-background-focus);
-    }
-
-    &--compact {
-        font-size: var(--ids-select-font-size-compact);
-        line-height: var(--ids-select-line-height-compact);
     }
 
     &--active,
     &.active {
+        background-color: var(--ids-select-background-active);
         border-color: var(--ids-select-border-color-active);
+
+        &:focus-visible,
+        &.focus {
+            background-color: var(--ids-select-background-active);
```

Comment:
is it showing the active border color when it has focus?

Reply:
doesn't look like it in the preview, I'd add the active border color here too. (Combobox too)